File: scripts/db-make.rb
require 'pg'
require 'json'
env = ENV['NODE_ENV']
if env.nil?
  env = 'development'
end
configfile = File.read("#{__dir__}/../config/dbconfig.json")
config = JSON.parse(configfile)
dbconfig = config["#{env}"]

dbConnection = PG.connect(dbname: 'postgres')
dbConnection.exec("CREATE DATABASE #{dbconfig["database"]}")
dbConnection.exec("CREATE USER #{dbconfig["username"]} WITH PASSWORD '#{dbconfig["password"]}'")
dbConnection.exec("GRANT ALL PRIVILEGES ON #{dbconfig["database"]} TO #{dbconfig["username"]}")

File: scripts/db-make.js
var pg = require("pg");
var env = process.env.NODE_ENV || 'development';
var dbconfig = require('../config/dbconfig.json')[env];

var pgurl = "postgres://localhost:5432/postgres";
var client = new pg.Client(pgurl);

client.connect(function(err){
    if(err){throw err;}

    client.query(`CREATE DATABASE ${dbconfig["database"]}`);
    client.query(`CREATE USER ${dbconfig["username"]} WITH PASSWORD '${dbconfig["password"]}'`);
    client.query(`GRANT ALL PRIVILEGES ON DATABASE "${dbconfig["database"]}" TO ${dbconfig["username"]}`);
})
process.exit();